docs/api/ipc-main.md

```diff
@@ -122,15 +122,15 @@ Handles a single `invoke`able IPC message, then removes the listener. See
 
 Removes any handler for `channel`, if present.
 
-## IpcMainEvent object
+## `IpcMainEvent` object
 
 The documentation for the `event` object passed to the `callback` can be found
-in the [`ipc-main-event`][ipc-main-event] structure docs.
+in the [`IpcMainEvent`][ipc-main-event] structure docs.
 
-## IpcMainInvokeEvent object
+## `IpcMainInvokeEvent` object
 
 The documentation for the `event` object passed to `handle` callbacks can be
-found in the [`ipc-main-invoke-event`][ipc-main-invoke-event]
+found in the [`IpcMainInvokeEvent`][ipc-main-invoke-event]
 structure docs.
```

Comment:
I wonder if we even need these anymore since the website has the ability to preview API structures and these structures are linked.

Reply:
Good point, it is a bit awkward having them as separate sections that basically just say follow the link. Could refactor to just make sure all usages of the event name are linked and drop these sections.

Reply:
@dsanders11 do you plan to change this or is this cleared to merge as-is?

Reply:
@codebytere, changed it per @erickzhao's suggestion.